src/wam2layers/tracking/backtrack.py

```diff
@@ -485,12 +485,12 @@ def run_experiment(config_file):
         # Determine the vertical moisture flux
         fluxes["f_vert"] = calculate_fv(fluxes, states_prev, states_next)
 
-        # Only track the precipitation at certain dates
+        # Only track the precipitation at certain timesteps
         if (
             time_in_range(
                 config.event_start_date,
                 config.event_end_date,
-                t.date(),
+                t,
             )
```

Comment:
I was just wondering if this time_in_range function now also works without the 'strftime'

Reply:
Yes it works with strftime but it was not needed